script.js

```diff
@@ -1,4 +1,259 @@
+// There will be only two players. One human and one computer (for the Base solution).
+// The computer will always be the dealer.
+// Each player gets dealt two cards to start. (both face up, dealer has 1 card open and 1 card closed)
+// The player goes first, and decides if they want to hit (draw a card) or stand (end their turn).
+// The dealer has to hit if their hand is below 17.
+// Each players' score is the total of their card ranks. Jacks/Queen/Kings are 10. Aces can be 1 or 11.
+// The player who is closer to, but not above 21 wins the hand.
+
+//global variables
+var deck = [];
+var gameMode = "first round";
+var winningMode = ""; //blackjack, bigger number max 21
+var playerHand = []; //to reset
+var computerHand = [];
+var gameRound = 1;
+
+function getRandomNumber(max) {
+  return Math.floor(Math.random() * max);
+}
+function shuffleDeck() {
+  for (i = 0; i < deck.length; i += 1) {
+    var randomIndex = getRandomNumber(deck.length);
+    var currentIndex = i;
+    var currentCard = deck[currentIndex];
+    var randomCard = deck[randomIndex];
+    deck[currentIndex] = randomCard;
+    deck[randomIndex] = currentCard;
+  }
+  return deck;
+}
+
+function getStartingHand(numberOfCards, array) {
+  for (i = 0; i < numberOfCards; i += 1) {
+    var card = dealCard();
+    array.push(card);
+  }
+  return array;
+}
+
+function dealCard() {
```

Comment:
This function is not that useful since it only pops something out of the deck. To make this function more reusable, you could consider having several parameters such as the number of cards that you would like to draw as well as whose hand to push the drawn card to